plugins/metrics/epochcommittment.go

```diff
@@ -0,0 +1,30 @@
+package metrics
+
+import (
+	"sync"
+
+	"github.com/iotaledger/hive.go/generics/event"
+
+	"github.com/iotaledger/goshimmer/packages/epoch"
+	"github.com/iotaledger/goshimmer/packages/notarization"
+)
+
+var (
+	lastCommittedEpoch      = epoch.NewECRecord(epoch.Index(0))
+	lastCommittedEpochMutex sync.RWMutex
```

Comment:
Can this be put in the epochstorage plugin as well? To have all the epochs stuff self-contained.

Reply:
The idea is to enable metrics plugin (grafana) without epochstorage plugin, so it's not necessary to store all the epoch contents unless we intended to.